modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -593,19 +593,19 @@ pairs to compute an initial homography estimate with a simple least-squares sche
 
 However, if not all of the point pairs ( \f$srcPoints_i\f$, \f$dstPoints_i\f$ ) fit the rigid perspective
 transformation (that is, there are some outliers), this initial estimate will be poor. In this case,
-you can use one of the three robust methods. The methods RANSAC, LMeDS and RHO try many different
+you can use one of the three robust methods. The methods RANSAC, LMEDS and RHO try many different
```

Comment:
In my opinion, when referring to the method we should keep `LMeDS`/`LMedS`.

See [Least Median of Squares](https://www-sop.inria.fr/odyssee/software/old_robotvis/Tutorial-Estim/node25.html#SECTION000105000000000000000) from a tutorial by Zhengyou Zhang on [Parameter Estimation Techniques: A Tutorial with Application to Conic Fitting](https://www-sop.inria.fr/odyssee/software/old_robotvis/Tutorial-Estim/Main.html), published in October 1995.